R/format_data.R

```diff
@@ -1413,7 +1416,7 @@ fmt_passthrough <- function(data,
         }
 
         x_str
-      },
+      }, # TODO: add rtf formatter
```

Comment:
Need to do this

Reply:
Now added along with a manual test RTF file.